### PR TITLE
Create Malaysian Language

### PR DESCRIPTION
**Pull request by ykris45**
Because the Malaysian language is similar to Indonesian, so I decided to add it to the firo website so that it can make the Malaysian people happy.

### CODE REVIEW

**Review by erciccione — COMMENTED** [review]
Thanks @ykris45! This looks good, but we need somebody to give a review to the translations before merging.

**Comment by ykris45** [comment]
Poke @justanwar as the President of Firo Site Repos

File: _i18n/ms.yml [hunk 1]
```diff
@@ -0,0 +1,304 @@
...
+  option3: pilihan cepat dan ringan
+  Qt: Qt Wallet
+  downloadqtw: muat turun QT Untuk Windows
+  downloadqtm: muat turun QT untuk MacOS
+  downloadqtl: muat turun QT Untuk Linux
+  electron: elektron
+  downloadelectronw: Muat Turun Electron Untuk Windows
+  downloadelectronm: Muat Turun Electron untuk MacOS
+  downloadelectronl: Muat Turun Electron Untuk Linux
+  elektrum: Elektrum
```
Comment (justanwar) [inline _i18n/ms.yml:101]: Do not change green things on the right, you'll break the translation.

File: _i18n/ms.yml [hunk 2]
```diff
@@ -0,0 +1,304 @@
...
+  downloadelectrumw: Muat Turun Electrum Untuk Windows
+  downloadelectrumm: Muat Turun Electrum untuk MacOS
+  downloadelectruml: Muat Turun Electrum Untuk Linux
+  hasheswindows: tunjukkan hash SHA256 Untuk Windows
+  hashesmac: tunjukkan hash SHA256 untuk MacOS
+  hasheslinux: tunjukkan hash SHA256 Untuk Linux
+  officialmobile: dompet mudah alih rasmi
+  optionmob1: Android
+  androidmob: Android Wallet
+  muat turunmobandroid: muat turun wallet
```
Comment (justanwar) [inline _i18n/ms.yml:111]: Same comments as Electrum.

File: _i18n/ms.yml [hunk 3]
```diff
@@ -0,0 +1,304 @@
...
+  campfiremob: dompet api unggun
+  downloadmobcamp: Muat Turun untuk Android
+  downloadioscamp: Muat turun untuk iOS
+  stackwalletmob: Stack Wallet
+  wallets: dompet pihak ketiga
+
+buyfiro:
+heading: Beli Firo
+heading2: salah satu cara untuk mendapatkan Firo adalah untuk memperolehnya di salah satu bursa. Di bawah ini anda akan dapati senarai pertukaran dan perkhidmatan pertukaran duit syiling yang menyokong Firo.
+predisclaimer: Penafian & kolon; & # x20;
```
Comment (justanwar) [inline _i18n/ms.yml:127]: Please fix

File: _i18n/ms.yml [hunk 4]
```diff
@@ -0,0 +1,304 @@
...
+penafian: sesetengah pautan ini mengandungi pautan rujukan di mana pasukan teras Firo mungkin menerima pampasan apabila anda mendaftar atau menggunakan produk/perkhidmatan mereka. Ini bukan merupakan pengesahan produk / perkhidmatan.
```
Comment (justanwar) [inline _i18n/ms.yml:128]: Same comments as Electrum.

File: _i18n/ms.yml [hunk 5]
```diff
@@ -0,0 +1,304 @@
...
+FF: "Firo / Fiat"
```
Comment (justanwar) [inline _i18n/ms.yml:129]: Same comments as Electrum.

File: _i18n/ms.yml [hunk 6]
```diff
@@ -0,0 +1,304 @@
...
+  auradesc: Aura adalah mekanisme pengundian peribadi kami untuk DAOs yang diilhamkan oleh teknologi Lelantus kami yang membolehkan pengundian tanpa nama tetapi boleh disahkan yang juga menyembunyikan kemajuan undi. Kami berhasrat untuk menggunakan ini sebagai asas untuk bergerak ke arah tadbir urus terdesentralisasi yang diformalkan.
+  avalanche: Avalanche
+  avalanchedesc: Avalanche adalah algoritma konsensus yang membolehkan transaksi segera, keselamatan yang dipertingkatkan dan peningkatan bebas garpu. Kami meneroka konsensus Avalanche sebagai pengganti potensi masternodes kami.
+  development: pembangunan
+  heading: Peta Jalan
+  mnodes: Masternodes deterministik (selesai)
+  mnodesq: Kuorum Masternode lama (selesai)
+  mtpstrip: jalur MTP (selesai)
+  mtpstripdesc: pelucutan MTP mengurangkan saiz blockchain Firo dari 60GB hingga 4GB dengan melucutkan bukti MTP dari blok lama.
+  penjenamaan semula: penjenamaan semula (selesai)
```
Comment (justanwar) [inline _i18n/ms.yml:247]: Same comments as Electrum.

**Comment by ykris45** [comment]
@justanwar re review

**Comment by justanwar** [comment]
Please do not tag me as I already get notified of everything in this PR.

A part of my review is still unresolved (consistent capitalization). For example, in English "Download QT for Windows" but the translation starts with a lowercase ("muat turun QT Untuk Windows")

**Pull request closed without merging**